src/data/faq_de.json

```diff
@@ -1342,6 +1342,35 @@
+                            "<b>Die Gesundheitsämter sind ausgelastet:</b><br/>Derzeit bewegen sich die COVID-19-Infektionzahlen auf sehr hohem Niveau und eine hohe Zahl von positiven Testergebnissen wird über die CWA übermittelt. Bitte haben Sie daher Verständnis, dass das Gesundheitsamt nicht alle Kontaktpersonen betreuen kann. Halten Sie dem Gesundheitsamt den Rücken frei für Aufgaben, die es nicht abgeben kann. Kontaktieren Sie Ihr Gesundheitsamt nur, wenn Sie unbedingt Unterstützung brauchen, z.B. weil Ihr Arbeitgeber oder Ihre Ausbildungsstätte eine Bescheinigung verlangt.",
+                            "<b>Handlungsempfehlungen:</b>",
+                            "<ul><li>Lassen Sie sich testen. Bei einer Warnung über ein erhöhtes Risiko („rote Kachel“) haben Sie Anspruch auf einen kostenlosen Test. Das muss nicht notwendigerweise ein PCR-Test sein.</li><li>Empfohlen sind mindestens 7 Tage, besser 10 Tage, für die Zeitdauer der Selbstquarantäne ab dem Tag der letzten Risikobegegnung. Das Datum der letzten Begegnung mit erhöhtem Risiko finden Sie in der roten Kachel.</li><li>Verzichten Sie nach Möglichkeit auf Besuche und Aktivitäten, bei denen Sie mit anderen Menschen zusammenkommen. Lassen Sie sich Lebensmittel liefern oder bitten Sie jemanden, die Einkäufe zu erledigen; meiden Sie öffentliche Verkehrsmittel und verzichten Sie auf Flug- und Zugreisen.</li><li>Vermeiden Sie insbesondere Kontakte zu vulnerablen Personen und Gruppen, beispielsweise zu Älteren und Vorerkrankten.</li><li>Minimieren Sie auch Kontakte im eigenen Haushalt.</li><li>Wenn Ihr Arbeitgeber Ihnen die Möglichkeit gibt, arbeiten Sie von zu Hause aus.</li><li>Achten Sie mindestens für 14 Tage (ab Risikobegegnung) auf Krankheitssymptome und wenden Sie sich an Ihre Hausarztpraxis, falls Sie sich krank fühlen oder Symptome auftreten sollten.</li><li>Halten Sie die Hygieneregeln ein.</li></ul>",
+                            "Weitere hilfreiche Informationen zur häuslichen Quarantäne finden Sie hier: <a href='https://www.rki.de/covid-19-quarantaene' target='_blank'>Häusliche Quarantäne (vom Gesundheitsamt angeordnet): Flyer für Kontaktpersonen</a><br/>Weitere Informationen zu Quarantäne-Maßnahmen finden Sie hier: <a href='#quarantine_measures'>https://www.coronawarn.app/de/faq/#quarantine_measures</a><br/>Testmöglichkeiten finden <a href='#where_can_i_get_tested'>Wo kann ich mich testen lassen?</a><br/>Weitere Informationen zum Testablauf: <a href='#red_card_how_to_test'>Rote Karte? So geht es zum Corona Test</a>"

```

Comment:
Similar comment here:

Instead of
"Weitere Informationen zu Quarantäne-Maßnahmen finden Sie hier: <a href='#quarantine_measures'>https://www.coronawarn.app/de/faq/#quarantine_measures</a>"

`Lesen Sie den Artikel <a href='#quarantine_measures'>Fragen zu Quarantäne-Maßnahmen</a> um weitere Details zu erfahren.`

Reply:
ok, thanks @MikeMcC399